.github/configs/quarterly-report-reminder.yaml

```diff
@@ -0,0 +1,258 @@
+buffer_window_days: 3
+projects:
+  - name: Hyperledger Iroha
+    github_org: hyperledger
+    github_repo: iroha
+    maintainers:
```

Comment:
Do you need to have separate maintainer lists? Seems like another place that will need to be updated.

Reply:
1. This parameter was introduced to tag specific maintainers, so that they are notified. It is optional.
2. I could not find a way to tag a team instead of individual maintainers. Also, I see there is inconsistency in the way `MAINTAINERS.md` file is generated across projects, making it difficult to pull the data through automation.

Reply:
@arsulegai https://github.com/daisy-row/ghe/blob/main/mup/cooked/maintainers.csv

Reply:
Thanks @ryjones , I will see if this file can be auto generated.

Reply:
just consume it for now

Reply:
The current PR is good to go until end of 2023. I will work on auto populating maintainers info, plus, consuming the reminder schedule file in later PRs.